pkgs/os-specific/linux/ffado/default.nix

```diff
@@ -63,23 +71,26 @@ mkDerivation rec {
     })
   ];
 
-  nativeBuildInputs = [
-    desktop-file-utils
-    scons
-    pkg-config
-    which
-    python
-    python3.pkgs.pyqt5
-    wrapQtAppsHook
-  ];
+  nativeBuildInputs =
+    [
+      desktop-file-utils
+      scons
+      pkg-config
+      which
+    ]
+    ++ lib.optionals withMixer [
+      python
+      python3.pkgs.pyqt5
+      wrapQtAppsHook
+    ];
 
   prefixKey = "PREFIX=";
   sconsFlags = [
     "DEBUG=False"
     "ENABLE_ALL=True"
     "BUILD_TESTS=True"
     "WILL_DEAL_WITH_XDG_MYSELF=True"
-    "BUILD_MIXER=True"
+    "BUILD_MIXER=${if withMixer then "True" else "False"}"
```

Comment:
Ideally, we would only build the mixer in the `mixer` package but it is not easy to prevent libffado from building.

And the mixer will try to launch `ffado-dbus-server` so we would need to be able to specify path to it from the libffado prefix.